src/cloud-api-adaptor/pkg/aa/config.go

```diff
@@ -1,51 +0,0 @@
-package aa
```

Comment:
Should this be kept so as to handle aa.toml via configMap ?

Reply:
@bpradipt configMap uses the whole aa.toml as a value, we don't need parse it anymore, refer https://github.com/confidential-containers/cloud-api-adaptor/blob/main/src/cloud-api-adaptor/docs/initdata.md